packages/components/src/SubHeaderBar/SubHeader.md

```diff
@@ -0,0 +1,31 @@
+# SubHeaderBar
```

Comment:
a link to the frontify spec would be awesome

Reply:
We are not allowed to do that. This is an internal tool, this repository is opensource